documentation/src/main/asciidoc/userguide/Bibliography.adoc

```diff
@@ -2,5 +2,5 @@
 
 [bibliography]
 - [[[PoEAA]]] Martin Fowler. https://www.martinfowler.com/books/eaa.html[Patterns of Enterprise Application Architecture].
-  Addison-Wesley Publishing Company. 2003.
-- [[[JPwH]]] Christian Bauer & Gavin King. https://www.manning.com/books/java-persistence-with-hibernate-second-edition[Java Persistence with Hibernate, Second Edition]. Manning Publications Co. 2015.
+  Addison-Wesley Professional. 2002.
+- [[[JPwH]]] Christian Bauer & Gavin King. https://www.manning.com/books/java-persistence-with-hibernate-second-edition[Java Persistence with Hibernate, Second Edition]. Manning. 2015.
```

Comment:
I changed the publishing companies as per Amazon book pages. The rationale to change both is due to inconsistency (one is `Publishing Company` whereas the other is `Publishing Co.`). I also found the publishing year for the former is wrong.